Makefile

```diff
@@ -210,8 +216,7 @@ travis-noop:
 .PHONY: travis-test-lint
 travis-test-lint: lint-python lint-js
 
-.PHONY: travis-test-sqlite travis-test-postgres travis-test-mysql travis-test-acceptance travis-test-snuba travis-test-symbolicator travis-test-js travis-test-cli travis-test-dist
-travis-test-sqlite: test-python
+.PHONY: travis-test-postgres travis-test-mysql travis-test-acceptance travis-test-snuba travis-test-symbolicator travis-test-js travis-test-cli travis-test-dist travis-test-riak
```

Comment:
Is there a `travis-test-sqlite` target that can be removed now?

Reply:
We remove that on the next line, not sure if I hadn't removed it before you left that comment.